fw/rbcx-coprocessor/include/Motor.hpp

```diff
@@ -71,50 +118,52 @@ class Motor {
         m_actualPosition = homedTicks;
     }
 
-    void setTargetPosition(int32_t ticks) {
-        if (m_state != PositionCtrl) {
+    void setTargetPosition(
+        const CoprocReq_MotorReq_SetPosition& req, bool additive) {
+        if (m_mode != MotorMode_POSITION) {
             m_positionReg.clear();
-            m_state = PositionCtrl;
+            modeChange(MotorMode_POSITION);
         }
-        m_targetPosition = ticks;
-    }
-
-    void addTargetPosition(int32_t ticksDelta) {
-        if (m_state != PositionCtrl) {
-            m_positionReg.clear();
-            m_state = PositionCtrl;
+        if (additive) {
+            m_targetPosition += req.targetPosition;
```

Comment:
Možná tu chceme `m_targetPosition = m_actualPosition + req.targetPosition;`? Takhle když pošlěš více requestů, tak se sčítají, přijde mi intuitivnější, když se ten předchozí "zruší".

Reply:
Domluveno, chcem zachovat. Lze použít home->setPosition pro použití aktuální pozice